GPSTest/src/main/java/com/chesapeaketechnology/gnssmonkey/AGpsMonkeyActivity.java

```diff
@@ -66,16 +69,18 @@ protected void onCreate(Bundle savedInstanceState) {
 //        openBatteryOptimizationDialogIfNeeded();
     }
 
+    /**
+     * Starts the GPS Monkey service which handles logging the GNSS data to a GeoPackage file.
+     */
     private void startService() {
         // TODO KMB: I don't think this case can happen. Even when Android was forced to kill the
         //  process (following instructions here: https://stackoverflow.com/a/18695974), when
         //  onCreate() was called, serviceBound was false and gpsMonkeyService was null.
         if (serviceBound) {
             gpsMonkeyService.start();
         } else {
-            startService(new Intent(this, GpsMonkeyService.class));
-            Intent intent = new Intent(this, GpsMonkeyService.class);
-            bindService(intent, serviceConnection, Context.BIND_AUTO_CREATE);
+            startService(serviceIntent);
+            bindService(serviceIntent, serviceConnection, Context.BIND_AUTO_CREATE);
```

Comment:
From my understanding (which is limited and could be wrong), calling bindService will also start the service.  However, once all the binders have been removed the service will be stopped.  If startService is called first instead, then once all binders are removed the service will continue running.  So really is is just a decision on the behavior of the service.

Reply:
This was existing code from @steveturner so I figured he had a reason for doing it this way. Certainly I think we want the service to stay running once our app is stopped in some cases, which is why he started it manually instead of implicitly through the bind call.

Reply:
This was existing code, but my guess is that @steveturner started the service first so that it could remain running when the app was stopped.